WEB_SERVICE/WEB/_CRITICAL/NOSQL_INJECTION/meta.json

```diff
@@ -0,0 +1,24 @@
+{
+  "risk_rating": "critical",
+  "short_description": "The database interaction in the application is susceptible to a NoSQL injection vulnerability.",
+  "references": {
+    "OWASP":"https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/07-Input_Validation_Testing/05.6-Testing_for_NoSQL_Injection"
+  },
+  "title": "NoSQL Injection",
+  "privacy_issue": true,
+  "security_issue": true,
+  "categories": {
+    "CWE_TOP_25": [
+      "CWE_918",
+      "CWE_943"
+    ],
+    "GDPR": [
+      "ART_5",
+      "ART_32"
+    ],
+    "PCI_STANDARDS": [
```

Comment:
I think "REQ_6_2" and "REQ_6_3", should be added as well:
![image](https://github.com/Ostorlab/KB/assets/144013278/9f0a3c18-950a-46bb-8bed-467d437dc5b0)